store/memory.go

```diff
@@ -120,7 +120,7 @@ func (e *MemStore) Get(ctx context.Context, commit []byte, domain eigendacommon.
 	}
 
 	// Don't need to do this really since it's a mock store
-	err = e.verifier.Verify(cert.BlobHeader.Commitment, encodedBlob)
+	err = e.verifier.VerifyCommitment(cert.BlobHeader.Commitment, encodedBlob)
```

Comment:
Observation, not a review comment The fact that there's a decent amount of code duplication between the EigenDA store and memory store suggests to me that we could do some refactoring.